backend/README.md

```diff
@@ -55,20 +55,28 @@ To update the username or password after it's been set:
 1. Login with the new credentials
 
 
-## Asynchronous execution/pipeline execution
+## Asynchronous Execution
 
- - Working with celery
- - Each pipeline or shared tasks will added to the queue (Redis), And the worker will consume from the queue
+This project uses Celery for handling asynchronous execution. Celery tasks are managed through various queues and consumed by workers.
+
+> ETL, TASK, and API Deployment are using these asynchronous workers. Log management also utilizes Celery.
+
+### Queues
+
+| Queue Name                 | Description                         |
+|----------------------------|-------------------------------------|
+| `celery`                   | Default queue for general Celery tasks. |
```

Comment:
NIT: @muhammad-ali-e can you elaborate on what these general tasks are? We could try to expand further on these queues with information from the confluence doc maybe

Reply:
Also are we even using this queue for anything?

Reply:
https://github.com/Zipstack/unstract/pull/648#discussion_r1743092356

Reply:
@ritwik-g If we not defined any specific queue for any task, The tasks are pushed to this queues , Currently, ETL,TASK and webhooks are using this queue . Let me are there any other than these items.